go.mod

```diff
@@ -57,6 +57,7 @@ require (
 	github.com/uber-go/tally v3.3.17+incompatible
 	github.com/uber/tchannel-go v1.19.0
 	github.com/urfave/cli v1.22.4
```

Comment:
can we replace v1? not suggesting in this PR but can we create a task for that?

Reply:
Yes, `tctl` still uses it. Will take care about it later.